src/components/ProfilePage/useAddressMenuContent.tsx

```diff
@@ -0,0 +1,97 @@
+import { useMenuStore } from '@/stores/menu';
+import { useAccount } from '@lifi/wallet-management';
+import LinkIcon from '@mui/icons-material/Link';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import ReceiptLongIcon from '@mui/icons-material/ReceiptLong';
+import { useTheme } from '@mui/material/styles';
+import { useRouter } from 'next/navigation';
+import { useTranslation } from 'react-i18next';
+import { TrackingAction, TrackingCategory } from 'src/const/trackingKeys';
+import { getSiteUrl, JUMPER_SCAN_PATH } from 'src/const/urls';
+import { useUserTracking } from 'src/hooks/userTracking';
+
+export const useAddressMenuContent = () => {
+  const { account } = useAccount();
+  const { trackEvent } = useUserTracking();
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const isDarkMode = theme.palette.mode === 'dark';
+  const closeAllMenus = useMenuStore((state) => state.closeAllMenus);
+  const router = useRouter();
+  const { setSnackbarState } = useMenuStore((state) => state);
+
+  const handleCopyButton = (textToCopy: string) => {
+    account?.address && navigator.clipboard.writeText(textToCopy);
+    setSnackbarState(true, t('navbar.walletMenu.copiedMsg'), 'success');
+  };
+
+  const handleScanButton = () => {
+    const url = `${JUMPER_SCAN_PATH}/wallet/${account.address}`;
+    trackEvent({
+      category: TrackingCategory.AddressMenu,
+      action: TrackingAction.OpenJumperScan,
+      label: 'open-jumper-scan-wallet',
+    });
+    router.push(url);
+  };
```

Comment:
Needs to be a link instead

Reply:
Ok for me as it is a menu and we reuse the same logic as `useMainMenuContent` no?